compose-table/src/test/java/org/dhis2/composetable/TableModelTest.kt

```diff
@@ -0,0 +1,87 @@
+package org.dhis2.composetable
+
+import org.dhis2.composetable.model.RowHeader
+import org.dhis2.composetable.model.TableCell
+import org.dhis2.composetable.model.TableHeader
+import org.dhis2.composetable.model.TableHeaderCell
+import org.dhis2.composetable.model.TableHeaderRow
+import org.dhis2.composetable.model.TableModel
+import org.dhis2.composetable.model.TableRowModel
+import org.dhis2.composetable.ui.TableSelection
+import org.junit.Test
+
+class TableModelTest {
+
+    private val tableModel = TableModel(
+        id = "table",
+        tableHeaderModel = TableHeader(
+            rows = listOf(
+                TableHeaderRow(
+                    cells = listOf(
+                        TableHeaderCell("1"),
+                        TableHeaderCell("2"),
+                        TableHeaderCell("3")
+                    )
+                )
+            )
```

Comment:
I think you can use a real model instead. It's already implemented

Reply:
I have checked it, but there is a specific test case that it's not in the model